app/views/lead_providers/school_details/show.html.erb

```diff
@@ -1,3 +1,104 @@
-<%= render ::SchoolInformation::Details::BreadcrumbComponent.new(selected_cohort: @selected_cohort) %>
-<%= render ::SchoolInformation::Details::HeadComponent.new(school: @school, selected_cohort: @selected_cohort) %>
-<%= render ::SchoolInformation::Details::MainInfoComponent.new(school: @school) %>
+<% content_for :title, @school.name %>
+<% content_for :before_content, govuk_breadcrumbs(breadcrumbs: {
+  "Schools" => lead_providers_your_schools_path,
+  @school.name => nil
+}) %>
+<% content_for :nav_bar do %>
+  <%= render PrimaryNavComponent.new do |component| %>
+    <%= component.nav_item(path: dashboard_path) do %>
+      Overview
+    <% end %>
+    <%= component.nav_item(path: lead_providers_your_schools_path, selected: true) do %>
+      Schools
+    <% end %>
+  <% end %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-6">
+      <%= @school.name %>
+    </h1>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-m"><%= @selected_cohort.start_year %> participants</h2>
+  </div>
+</div>
+
+<div class="govuk-grid-row govuk-!-margin-bottom-6">
+  <div class="govuk-grid-column-one-third">
+    <div class="dashboard-numbers key-number">
+      <p class="govuk-heading-xl">0</p>
```

Comment:
Why 0 here and the correct number in the previous screen?

Reply:
Doh!